tests/conftest.py

```
@@ -1,10 +1,95 @@
+def review_manager(db_manager, fsrs_scheduler):
+    """Fixture to provide a ReviewSessionManager instance."""
+    # Provide dummy UUIDs for user and deck, as required by the new constructor.
+    # This is a common practice in tests where the specific UUID value isn't critical.
+    user_uuid = uuid4()
+    deck_uuid = uuid4()
+    return ReviewSessionManager(
+        db_manager=db_manager,
+        scheduler=fsrs_scheduler,
+        user_uuid=user_uuid,
+        deck_uuid=deck_uuid,
+    )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'deck_uuid' in constructor call

(E1123)